runtime/dotnet/azurewebapp/Startup.cs

```diff
@@ -132,7 +132,11 @@ public void ConfigureServices(IServiceCollection services)
             // Register AuthConfiguration to enable custom claim validation for skills.
             if (IsSkill(settings))
             {
-                services.AddSingleton(sp => new AuthenticationConfiguration { ClaimsValidator = new AllowedCallersClaimsValidator(settings.Skill) });
+                services.AddSingleton(sp => new AuthenticationConfiguration { ClaimsValidator = new AllowedCallersClaimsValidator(settings.SkillConfig) });
```

Comment:
Same comment as above
```suggestion
                services.AddSingleton(sp => new AuthenticationConfiguration { ClaimsValidator = new AllowedCallersClaimsValidator(settings.SkillConfiguration) });
```